CONTRIBUTING.md

```diff
@@ -27,39 +27,18 @@ or that there are particular issues that you should know about before implementi
 
 We aim to maintain high code quality, therefore every PR goes through the same review process, no matter who opened the PR.
 
-### General advice
```

Comment:
No change in this part - I just move this to the end, because I feel it's less important.

Reply:
Thanks for noting this, it saves me a bunch of time 😅